_internals/system-views.md

```diff
@@ -4,47 +4,47 @@ title: Babelfish system view
 nav_order: 4
 ---
 
-## Babelfish system views 
+## Babelfish system views
 
-Once in a while, an application has to access an MS SQL-compatible system view to
+Once in a while, an application has to access an MS SQL-compatible system view
+or a server side function to
 work properly. This is especially true for graphical user interfaces. However,
-accessing an MS SQL-compatible system view is also necessary for some applications 
+accessing an MS SQL-compatible system view is also necessary for some applications
```

Comment:
MS SQL is not a product. Use correct name

Reply:
You are right; changed.

Reply:
This page also refers to MS SQL: https://babelfishpg.org/docs/vision/

It also has some grammar that could use updating - I'm mentioning it here because I've not seen a pull request for that page, or for: https://babelfishpg.org/docs/contributing/

Reply:
@susanmdouglas-aws Yes, but that has nothing to do with this pull request, does it? Could you open an issue and assign it to me?

Reply:
:) Sorry, haven't used Github in ages... opened #89...